docs/003-onchain-monitoring.adoc

```diff
@@ -0,0 +1,28 @@
+= ADR 003 - Onchain monitoring
+Richard Holzeis <richard@10101.finance>
+3.0, July 29, 2022: AsciiDoc article template
+:toc:
+:icons: font
+:attributes: 2023-01-31
+
+This document outlines the decision how onchain transactions are monitored.
+
+== Decision
+
+10101 will use a public electrum server to monitor onchain transactions. Electrum is broadly accepted and used. We will use public infrastructure as the trust model will have no impact to the user whether or not 10101 would run the infrastructure components or delegates that task to public infrastructure. A corresponding setting to the user to choose her infrastructural components could be added within the app.
```

Comment:
```suggestion
10101 will use a public electrum server to monitor onchain transactions. Electrum is broadly accepted and used. We will use public infrastructure as the trust model will have no impact to the user whether or not 10101 would run the infrastructure components or delegates that task to public infrastructure. A corresponding setting to the user to choose her infrastructural components will be added within the app to allow the user to be fully self-sovereign.
```

I would be explicit here and I would add this from the get-go.
In the long run it might become less important, but for our initial target users I think it is, and it should be a low-hanging fruit!